share/spice/transit/switzerland/transit_switzerland.js

```diff
@@ -0,0 +1,58 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_transit_switzerland = function(api_result){
+
+        if (api_result.error || !api_result.to || !api_result.from || !api_result.connections) {
+            return Spice.failed('transit_switzerland');
```

Comment:
The argument here should match the `id` field of your `Spice.add()` (so change it to just `'switzerland'`)